docs/environment-variables.md

```diff
@@ -70,13 +70,32 @@ export KEY=VALUE
 export FLASK_ENVIRONMENT_CONFIG=dev
 ```
 
-Windows user should use **set** instead of **export** for setting these environment variables. <br/>
 Another way to do this in flask applications is by having a file called `.env` which will have all of the environment variables. When a flask application runs, it will load these variables from the file.
-
 - Content of `.env` file:
-
 ```
 FLASK_ENVIRONMENT_CONFIG=dev
 SECRET_KEY='some_random_key'
 (...)
 ```
+
+# Database Enviorment Variables
+
+To run the backend in other than `local` mode i.e `prod` or `dev` you need to export database environment variables.
+These are the required database environment variables.
+```
+export DB_TYPE=<database_type>
+export DB_USERNAME=<database_username>
+export DB_PASSWORD=<database_password>
+export DB_ENDPOINT=<database_endpoint>
+export DB_NAME=<database_name> |
+```
+ ## Database environment variables description
+
+
+| Environment Variable | Description                                                                 | Example           |
+|----------------------|-----------------------------------------------------------------------------|-------------------|
+| DB_TYPE | Type of database you want to use | postgres, mysql |
+| DB_USERNAME          | Username of the user through which you will be doing operations in database | admin123, jhonDoe | 
```

Comment:
```suggestion
| DB_USERNAME          | Username of the user through which you will be doing operations in database | admin123 | 
```

Just one example here is ok :) so that it does not confuse contributors into thinking they have to put 2 values